fetch.js

```diff
@@ -24,14 +24,14 @@ function channelMap(channel, file) {
     aurora:  file ? 'latest-aurora'  : 'firefox-aurora-latest',
     nightly: file ? 'latest-nightly' : 'firefox-nightly-latest'
   }
-  return map[channel]
+  return map[channel] || 'firefox-' + channel;
```

Comment:
This allows the caller to pass in an arbitrary `channel` like `48.0` or `46.0.1`. If it's not on the list, no problem.